screens/AddHotspotScreen.tsx

```diff
@@ -184,23 +187,28 @@ const getStyles = (theme: ReactNativePaper.Theme) =>
     },
   });
 
-export const AddHotspotScreen = () => {
+export const AddHotspotScreen = ({
+  route,
+}: RootStackScreenProps<'AddHotspot'>) => {
+  const navigation = useNavigation();
   const theme = useTheme();
   const styles = getStyles(theme);
 
   const [location, setLocation] = useState<Location>();
+  const [name, setName] = useState<string>('');
```

Comment:
minor: specifying the type for the generic `useState` is redundant if you provide the default value `''`
`const [name, setName] = useState('');` this is enough

Reply:
done.